project_closing/models/project.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.multi
+    def toggle_active(self):
+        for record in self:
+            record.analytic_account_id.active = not record.active
```

Comment:
@espo-tony I took a better look at this, and I agree with your comment. I can do it directly in this PR.